Jamulus.pro

```diff
@@ -229,18 +229,42 @@ win32 {
     LIBS += -framework AVFoundation \
         -framework AudioToolbox
 } else:android {
-    ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+    # ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+    # note: only armeabi-v7a arm64-v8a are targeted, others are dead/deprecated
```

Comment:
x86_64 should be supported.

Reply:
Why do you need x86_64? There are almost zero x86_64 Android devices.
This CI is to build a package for submission to the Play Store, for installation on real-world devices.

And even for local building and testing, I have found it infinitely superior to test with armv8 images in the simulator, or to connect Android devices directly - it reduces the error "surface area" a lot.

Reply:
Ok, I've thought of a good possible reason to build x86_84 ... :)

The Play Store submission will also work on Chromebooks - even if Intel-based (this is tested and working). But that will involve ARM->Intel translation which will likely slow performance.

I'll look at adding x86_64 support in ASAP.

Reply:
In fact, since Chromebook support is basically "out-of-the-box" with the Play Store submission process, there's an argument for including plain old x86 as well - at least according to https://developer.android.com/topic/arc/device-support ...

Reply:
Yes, and also WSA (at least for testing purposes). I still own an Intel powered tablet...

Reply:
Good point on WSA!
I've now updated the build to target all the 4 x ABIs (including x86 and x86_64), and all are submitted to Play Store as part of the same release. (Tested and working - see the Koord app in Play Store for example.)